homeassistant/components/roon/media_player.py

```diff
@@ -163,6 +163,76 @@ def update_data(self, player_data=None):
 
     def update_state(self):
         """Update the power state and player state."""
+
+        def _parse_volume(player_data):
```

Comment:
think yiu could make this a class method rather than nested since its not really dynamicly needed

Reply:
I switched this to a class method, and `_parse_now_playing` to a standard method since it needs access to the api instance variable.